ssh-keygen.c

```diff
@@ -3511,6 +3532,8 @@ main(int argc, char **argv)
 		for (i = 0; i < nopts; i++) {
 			if (strcasecmp(opts[i], "no-touch-required") == 0) {
 				sk_flags &= ~SSH_SK_USER_PRESENCE_REQD;
+			} else if (strcasecmp(opts[i], "require-pin") == 0) {
```

Comment:
should this be user-verification-required, too?

Reply:
I made it verify-required to match auth-options.c